lib/phoenix/digester.ex

```diff
@@ -294,6 +298,36 @@ defmodule Phoenix.Digester do
     end
   end
 
+  @doc """
+  Deletes compiled/compressed asset files, including the cache manifest.
+
+  ## Arguments
+
+    * `path` - The path where the compiled/compressed files are saved
+
+  """
+  @spec clean_all(String.t()) :: :ok | {:error, :invalid_path}
+  def clean_all(path) do
+    if File.exists?(path) do
+      %{"digests" => digests} = load_manifest(path)
```

Comment:
I was wondering if implementation wise we could simply call `clean()` with age of 0 and keep of 0? And the additional `--compressed` flag? But if this is simpler, then it is good for me too!